src/decode.h

```diff
@@ -351,6 +351,14 @@ typedef struct PktProfiling_ {
 /* forward declartion since Packet struct definition requires this */
 struct PacketQueue_;
 
+/*
+   since a pkt could come from different runmodes, we need to set
+   a mode to be able to do an action later.
+   (e.g. if a pkt come from NFQ, pkt_mode will be set to IPS and
+         it means that we'll be able to drop it.)
+*/
+enum PktMode {IDS, IPS, AUTO};
```

Comment:
I think it's better to call the members something like PKT_MODE_IDS, PKT_MODE_IPS, PKT_MODE_AUTO